src/renderer/routes/test_sequencer_panel/components/DesignBar.tsx

```diff
@@ -165,11 +222,11 @@ export function countWhere<T>(arr: T[], predicate: (t: T) => boolean): number {
   return arr.reduce((acc, t) => (predicate(t) ? acc + 1 : acc), 0);
 }
 
-const getNumberOfTest = (data: TestSequenceElement[]): number => {
+const getNumberOfTestInSeq = (data: TestSequenceElement[]): number => {
```

Comment:
`return _.reduce(data, (acc, cur) => (cur.type === "test" ? acc + 1 : acc), 0);`

Reply:
you can get rid of the `countWhere` function and use https://lodash.com/docs/4.17.15#reduce

Reply:
lodash super convenient

Reply:
`countWhere` literally just does the reduce snippet you posted there lol

Reply:
yep they do the same, we can just use the lodash one instead of writing the same func again

Reply:
having to write the logic to accumulate the sum over the list is annoying, the point of `countWhere` is that you can just pass a predicate

Reply:
oh I see, nvm!